app/models/featured_content.rb

```diff
@@ -1,17 +1,11 @@
+# frozen_string_literal: true
+
 class FeaturedContent < ApplicationRecord
-  belongs_to :manifestation
-  belongs_to :authority, inverse_of: :featured_contents
+  belongs_to :manifestation, optional: true
+  belongs_to :authority, inverse_of: :featured_contents, optional: true
   belongs_to :user
+  validates :title, :body, presence: true
 
   # attr_accessible :title, :body, :external_link
   has_many :featured_content_features, class_name: 'FeaturedContentFeature', dependent: :destroy
-
-  def featured_list
```

Comment:
moved this method to haml view. I believe it is better to have such formatting logic in view